app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,7 @@
     <uses-permission android:name="android.permission.WAKE_LOCK"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
This permission has to be requested at runtime. I re-installed the current `develop` branch on an emulator and notifications are disabled now.

Reply:
Oh, oops. I only tested upgrading, no fresh install.